LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,16 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail MIQ Logs:
+function tailmiq()
+{
+  tail -f /var/www/miq/vmdb/log/$1.log
+}
+alias tailpglog='tail -f /var/opt/rh/rh-postgresql95/lib/pgsql/data/pg_log/postgresql.log'
+
+# Rails Console:
+alias railsc="cd /var/www/miq/vmdb;echo '\$evm = MiqAeMethodService::MiqAeService.new(MiqAeEngine::MiqAeWorkspaceRuntime.new)'; rails c"
```

Comment:
Yes, as @jeffwarnica said, we need to use bin/rails.

Also, do you always want to inject a MiqAeWorkSpaceRuntime into the generic sounding rails console?  
If so, maybe we should have a different name so it's clearly for automate.

An alternative is a console method you can call in console that is as an alias for setting up `$evm` as a workspace.

[Console methods live here](https://github.com/ManageIQ/manageiq/blob/3923a425262528a60fc1d65c3ddb82ebb9f4cac1/lib/vmdb/console_methods.rb#L3-L32) and already get loaded when running rails console.  cc @mkanoor @gmcculloug